ui/lib/core/addon/templates/components/replication-mode-summary.hbs

```diff
@@ -105,9 +105,9 @@
         Enable
       {{/link-to}}
     {{else if (eq mode 'dr')}}
-      {{cluster.drReplicationStateDisplay}}
+      {{cluster.drStateDisplay}}
```

Comment:
hey @meirish, i wanted to run this change by you. i was trying to see where `stateDisplay` from the [cluster model](https://github.com/hashicorp/vault/blob/c5efd226eda702217b5b18edcf42cc215e9e32eb/ui/app/models/cluster.js#L52) was use, and i couldn't find any evidence of it actually being used in the codebase. the closest thing i could find was this template which called `drReplicationStateDisplay`. i presume this was a typo?

similarly, do you happen to know where in the UI this code is actually executed? i want to make a large refactor to `stateDisplay` without breaking anything, but i haven't been able to find it from clicking around. 

thanks for your help!

Reply:
Sorry for the late reply! I think `drReplicationStateDisplay` may be something that was missed in the first refactor of the replication UI after @joshuaogle joined. 

Might be too late to be helpful, but for the second question - do you mean where `replication-mode-summary` is used? I think it's still using old ember component syntax https://github.com/hashicorp/vault/search?q=replication-mode-summary&unscoped_q=replication-mode-summary shows it in replication box in the user menu, and then in `replication-summary` which I think is used on the top level replication page where you can enable Perf or DR replication (I'm not sure that page exists as it did with the rework happening though so it may not be relevant).